basictracer/propagation.py

```diff
@@ -58,8 +58,8 @@ def __init__(self, tracer):
         self.tracer = tracer
 
     def inject(self, span_context, carrier):
-        carrier[field_name_trace_id] = '{:x}'.format(span_context.trace_id)
-        carrier[field_name_span_id] = '{:x}'.format(span_context.span_id)
+        carrier[field_name_trace_id] = '{0:x}'.format(span_context.trace_id)
```

Comment:
@bg451 can't this just be `'{0}'`? Or, alternatively, `'{x}'.format(x=span_context.trace_id)`...